nala/utils/nala.run.js

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env node
```

Comment:
we don't have to change it in this PR but i find having both `pr.run.sh` and `nala.run.js` doing similar thing confusing, are we sure we need them both? or could be 1 script?